client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	makeTour,
+	Quit,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import {
+	// hasUserRegisteredBefore,
+	// isAbTestInVariant,
+	isEnabled,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+import Button from 'components/button';
+
+export const BoostedPostTour = makeTour(
+	<Tour
+		name="boostedPost"
+		path="/post/"
+		version="20170120"
+		when={ and(
+			isEnabled( 'boosted-post' ),
+			isDesktop,
+			// hasUserRegisteredBefore( new Date( '2016-12-15' ) ),
+			// isAbTestInVariant( 'boostedPostTour', 'enabled' )
+		) }
+	>
+		<Step name="init"
+			target=".editor-ground-control__publish-combo"
+			arrow="left-top"
+			placement="beside"
```

Comment:
You'll probably also want `scrollContainer=".sidebar__region"`, and potentially `shouldScrollTo`. See other tours and how steps react when the user scrolls down the page.